### PR TITLE
feat: Do not save event to Postgres

### PR DESCRIPTION
**Pull request by lynnagara**
Since we fetch events exclusively from Snuba now, we no longer need to
write events to Postgres.

Saving can be (temporarily) restored for rollout by setting `store.skip-pg-save`
to False.

### CODE REVIEW

**Comment by lynnagara** [comment]
Not really ready for review, just trying to fix CI

File: src/sentry/event_manager.py [hunk 1]
```diff
@@ -753,26 +719,25 @@ def save(self, project_id, raw=False, assume_normalized=False):
...
-            )
+        event.data.save()
 
-            if CHECK_DUPLICATES:
-                return event
+        if not options.get("store.skip-pg-save", False):
+            # save the event
+            try:
+                with transaction.atomic(using=router.db_for_write(Event)):
+                    event.data.save()
```
Comment (fpacifici) [inline src/sentry/event_manager.py]: This way you are calling event.data.save() twice as long as postgres save is not disabled. Is it intended ?

Comment (lynnagara) [reply src/sentry/event_manager.py]: No definitely not, been meaning to fix this. Thanks.

**Comment by lynnagara** [comment]
Note: we need to make sure "store.skip-pg-save" is set to False in prod first

**Comment by lynnagara** [comment]
Option has now been set to False in prod

**Review by fpacifici — APPROVED** [review]
please check the table is actually not read in other cases than deletion before turning the flag on.

**Comment by lynnagara** [comment]
Looks to be just deletions so far. Going to merge then test this.

**Pull request merged**